src/utils/date-utils-test.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2020 The Subscribe with Google Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {toTimestamp} from './date-utils';
+
+/**
+ *
+ * @param {*} val
+ * @return {boolean}
+ */
+function isInteger(val) {
+  return Math.floor(val) === Math.ceil(val);
+}
+
+describe('toTimestamp', () => {
+  it('should create a timestamp', () => {
+    const date = new Date();
```

Comment:
To make this test more readable, could you use a consistent seed for the Date? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date

This would allow for simple values in the expectations instead of needing to repeat equations in the test

Reply:
sure